src/recordlinker/schemas/pii.py

```diff
@@ -234,36 +234,31 @@ class Telecom(StrippedBaseModel):
     system: typing.Optional[str] = None
     use: typing.Optional[str] = None
 
-    def phone(self) -> str | None:
-        """
-        Return the phone number from the telecom record.
-        """
-        if self.system != "phone":
-            return None
-        # normalize the number to include just the 10 digits
-        return re.sub(r"\D", "", self.value).strip()[:10]
-
-    def email(self) -> str | None:
-        """
-        Return the email address from the telecom record.
-        """
-        if self.system != "email":
-            return None
-        return self.value.lower().strip()
-
-    @classmethod
-    @functools.lru_cache()
-    def get_system_handlers(cls) -> dict[str, str]:
-        """
-        Return a dictionary of system handlers for the Telecom class where the keys
-        are the system values and the values are the method names to call.
-
-        """
-        return {
-            name: name
-            for name, method in cls.__dict__.items()
-            if callable(method) and not name.startswith("_")
-        }
+    @pydantic.model_validator(mode="before")
+    def validate_and_normalize_telecom(cls, values):
+        """
+        Validate and normalize the telecom record.
+        """
+        # If telecom.system = "email", set telecom.value to lowercase
```

Comment:
Based on the comment, I think we're missing a call to `lower()`

Reply:
Oops, you're right